templates/confirmation/confirm_email_change.html

```diff
@@ -9,8 +9,10 @@ <h1 class="get-started">{{ _('Email changed!') }}</h1>
         </div>
 
         <div class="app-main white-box">
+            {% trans %}
```

Comment:
Can we use the nicer variable syntax so that we can just provide the full <a> tags as variables so it's just "from {old_email_str} to {old_email_str}" for translators?  Like you did with the email templates.

Reply:
@timabbott I actually thought about that when I was adding the tags and realized that we have not done something like this for Jinja templates. I think you are probably referring to how I did the markdown links for the welcome bot.

Anyway, I have added a commit does this. If that looks good to you feel free to merge them both as a single commit.